### PR TITLE
Updates Cloud Native eBook to Net 5

### PR DESCRIPTION
**Pull request by sughosneo**
This PR updates the following pages :

- `Index`
- `Introduction to cloud-native applications`
- `Candidate apps for cloud native`
- `Introducing eShopOnContainers reference app`
- `Mapping eShopOnContainers to Azure Services`
- `Deploying eShopOnContainers to Azure`
- `Centralized configuration`
- `Defining Cloud Native`
- `Introducing eShopOnContainers reference app`
- `Mapping eShopOnContainers to Azure Services`
- `Leveraging containers and orchestrators`
- `Front-end client communication`
- `Service-to-service communication`
- `Distributed data`
- `Relational vs. NoSQL data`
- `Application resiliency patterns`
- `Observability patterns`
- `Logging with Elastic Stack`
- `Monitoring in Azure Kubernetes Services`
- `Azure Monitor`
- `Identity`
- `Azure security for cloud-native apps`
- `DevOps`
- `Cloud Native Application Bundles`

### CODE REVIEW

File: docs/architecture/cloud-native/application-bundles.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 
-The Docker containers may run on Kubernetes using a Helm Chart for deployment. The Azure Functions may be allocated using Terraform templates. Finally, the virtual machines may be allocated using Terraform but built out using Ansible. This is a whole mess of technologies and there has been no way to package them all together into a reasonable package. Until now.
+The Docker containers may run on Kubernetes using a Helm Chart for deployment. The Azure Functions may be allocated using Terraform templates. Finally, the virtual machines may be allocated using Terraform but built out using Ansible. This aspect is a whole mess of technologies and there has been no way to package them all together into a reasonable package. Until now.
```
Comment (robvet) [inline docs/architecture/cloud-native/application-bundles.md:11]: "Whole mess"??  Maybe, a "large variety" might be more appropriate.

Comment (sughosneo) [reply docs/architecture/cloud-native/application-bundles.md]: Thanks @robvet. Yes, that would be better. Included it in the latest commit.

**Comment by sughosneo** [comment]
Hi @IEvangelist - if you could review this PR and share your feedback as well. Thank you.

File: docs/architecture/cloud-native/index.md [hunk 2]
```diff
@@ -54,14 +54,16 @@ Participants and Reviewers:
 > **Jeremy Likness**, Senior Program Manager, .NET team, Microsoft
 >
 > **Cecil Phillip**, Senior Cloud Advocate, Microsoft
+>
+> **Sumit Ghosh**, Principal Consultant at Neudesic
 
 Editors:
 
 > **Maira Wenzel**, Program Manager, .NET team, Microsoft
```
Comment (IEvangelist) [inline docs/architecture/cloud-native/index.md:62]:
```suggestion
Editors:

> **Maira Wenzel**, Program Manager, .NET team, Microsoft

> **David Pine**, Senior Content Developer, .NET docs, Microsoft
```

**Review by IEvangelist — APPROVED** [review]

**Review by nishanil — APPROVED** [review]
Looks good to me

**Pull request merged**